src/gateways/accountHolderApiGateway.ts

```diff
@@ -0,0 +1,25 @@
+import axios, { AxiosResponse } from "axios";
+import { IAccountHolderIdResponseBody } from "../lib/accountHolder/accountHolderIdResponseBody";
+
+export class AccountHolderApiGateway {
+  private readonly apiUrl: string;
+  private readonly accountHolderIdEndpoint = "account-holder/auth-id";
+
+  constructor() {
+    this.apiUrl = process.env.API_URL;
+  }
+
+  public async getAccountHolderId(
+    authId: string,
+    accessToken: string
+  ): Promise<string> {
+    const url = `${this.apiUrl}/${this.accountHolderIdEndpoint}/${authId}`;
+    let response = await axios.get<
```

Comment:
```suggestion
    const response = await axios.get<
```

To fix the linter :-)

In the JS world it's conventional (at least, I think...) to reach for `const` first and avoid `let/var`ing a variable unless you absolutely must.  There's a sensible aversion to mutation that is conspicuously absent in Java/C#land 😝

Reply:
done